Fix an issue where randomizer/pack frames don't open

hwjsnc opened the pull request:
Fixes a crash that happens when opening the randomizer/pack frame
if no unpacked data is available. Instead, the pack/randomize buttons
are disabled in that case.

nnguyen259 commented on frames/randomizer.py:
```diff
@@ -47,6 +48,8 @@ def __init__(self, master=None):
 
         self.btnRandomize = ttk.Button(self, text='Randomize', command=self.doRandomize)
         self.btnRandomize.grid(column=2, row=5, padx=5, pady=5, sticky='nwse')
+        if not projects:
```
Could maybe move the `self.entryName.current(0)` down here then wrap it in a try block? Then disable the button if it causes an exception and maybe display a popup that says no project or something?

Same for the pack frame

hwjsnc replied:
I'm not sure causing and catching an exception is a good idea here because I can't find any documentation on it? It's a `_tkinter.TclError`, so it's probably internal. Showing a message box seems like a good idea. How about this:

```
        if projects:
            self.entryName.current(0)
        else:
            messagebox.showerror('No Data', 'No project data available. Please run the unpacker first.')
            raise RuntimeError("no project data available") # prevents the frame from changing
```

nnguyen259 commented:
hmm maybe we should catch it further, like out on the main screen. When someone click the pack/randomize button we check the project folder, if there isn't any we display a message and stop the screen change

hwjsnc commented:
I moved the check to mainFrame now.

hwjsnc committed:
Fix an issue where randomizer/pack frames don't open

When the projects folder is empty, the packer and randomizer frames
don't work. With this patch, if a user attempts to open them, an error
message is shown.
